lib/cartopy/mpl/slippy_image_artist.py

```diff
@@ -41,6 +41,20 @@ def __init__(self, ax, raster_source, **kwargs):
         self.raster_source = raster_source
         super(SlippyImageArtist, self).__init__(ax, **kwargs)
         self.set_clip_path(ax.outline_patch)
+        self.cache = []
+
+        ax.figure.canvas.mpl_connect('button_press_event', self.on_press)
+        ax.figure.canvas.mpl_connect('button_release_event', self.on_release)
+
+        self.on_release()
+
+    def on_press(self, event=None):
+        self.user_is_interacting = True
+        self.stale = True
```

Comment:
Why does a mousedown trigger a redraw?

Reply:
You know, that is a fine question that I didn't think to ask.
I can't see any benefit to it either.

Reply:
#1197